tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1093,7 +1093,19 @@ TEST (nnstreamer_capi_switch, dummy_01)
   EXPECT_EQ (status, ML_ERROR_NONE);
   wait_for_start (handle, state, status);
 
-  g_usleep (300000); /* 300ms. Let a few frames flow. */
+  /* Waiting for 3 frames to arrive */
+  while (*count_sink < 3) {
+    timeout_count++;
+    g_usleep (10000);
+    if (timeout_count > 100) {
+      EXPECT_TRUE (false);
```

Comment:
@myungjoo @jaeyun-jung 
Timeout occurs here in the arm gbs build. Is one second really not enough? Or is there another problem?